src/start.test.js

```diff
@@ -0,0 +1,31 @@
+import { startServer } from './start'
+import supertest from 'supertest'
+import logger from 'loglevel'
+
+const port = 3002
+let server
+
+beforeEach(async () => {
+  server = await startServer({ port })
+})
+
+afterEach(async () => {
+  await server.close()
+})
```

Comment:
In teoria, se stai seguendo la struttura dei describe, questi due andrebbero nel blocco di codice del describe corrispondente, a meno che non vuoi che questi lifecycle vengano triggerati in tutti i describe potenziali di questo test 🤔